docker-compose/technical/grafana/config/datasources/datasource.yml

```
@@ -0,0 +1,45 @@
+apiVersion: 1
+
+datasources:
+    - name: Prometheus
+      type: prometheus
+      access: proxy
+      url: http://172.17.0.1:8000
```

prefer network container name to not have to manage the IP, and not having to pass by host port
```suggestion
      url: http://prometheus:9090
```

---

Done